using_images/xpaas_images/eap.adoc

```diff
@@ -9,7 +9,94 @@
+
+This topic details the differences between the JBoss EAP xPaaS image and the regular release of JBoss EAP, and provides instructions specific to running and configuring the JBoss EAP xPaaS image. Documentation for other JBoss EAP functionality not specific to the JBoss EAP xPaaS image can be found in the  https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/[JBoss EAP documentation on the Red Hat Customer Portal].
+
+The location of `_EAP_HOME_` inside a JBoss EAP xPaaS image is: *_/opt/eap/_*. 
+
+== Differences Between the JBoss EAP xPaaS Image and the Regular Release of JBoss EAP
+
+=== Functionality Differences for OpenShift JBoss EAP xPaaS Images
+
+There are several major functionality differences in the OpenShift JBoss EAP xPaaS image:
+
+* The JBoss EAP Management Console is not available link:#Managing-OpenShift-JBoss-EAP-xPaaS-Images[to manage OpenShift JBoss EAP xPaaS images].
+* The JBoss EAP Management CLI is only bound locally. This means that you can only access the Management CLI of a container from within the pod. For example: 
++
+----
+oc exec p <pod> -c <container> - /opt/eap/bin/jboss-cli.sh
+----
++
+[WARNING]
+Any configuration changes made using the JBoss EAP Management CLI on a running container is not persistent. 
+* Domain mode is not supported. OpenShift controls the creation and distribution of applications in the containers.
+* There is no HornetQ messaging subsystem. The JBoss A-MQ xPaaS image can be used for scenarios that require messaging. 
+
+[[Managing-OpenShift-JBoss-EAP-xPaaS-Images]]
+=== Managing OpenShift JBoss EAP xPaaS Images
+
+A major difference in managing an OpenShift JBoss EAP xPaaS image is that there is no Management Console exposed for the JBoss EAP installation inside the image.
+
+As containers are not persistent, the JBoss EAP Management Console is not included. However, the JBoss EAP Management CLI (*_EAP_HOME/bin/jboss-cli.sh_*) is still accessible from within the container for troubleshooting purposes, however any configuration changes made on a running OpenShift container will be lost when the container is destroyed.
+
+link:#Making-Configuration-Changes-EAP[Making configuration changes to the JBoss EAP instance inside the JBoss EAP xPaaS image] is different to the process you may be used to for a regular release of JBoss EAP. 
+
+=== Unsupported Configurations
+
+The following is a list of unsupported configurations specific to the JBoss EAP xPaaS image:
+
+* Using MySQL in a scaled environment with XA distributed transactions is not recommended. For applications that support both scaling and XA distributed transactions, PostgreSQL is recommended instead. 
+// This is based on https://issues.jboss.org/browse/CLOUD-56
+
+== Installing the JBoss EAP xPaaS Image Streams and Application Templates
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must first install the image streams and source-to-image (S2I) application templates.
```

Comment:
s/source-to-image/Source-to-Image/

Per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#source-to-image-s2i